.github/workflows/dart_ci.yml

```diff
@@ -2,9 +2,12 @@ name: Dart CI
 
 on:
   push:
-    branches: [ master ]
```

Comment:
These changes are just merged in from https://github.com/Workiva/over_react/pull/670